invenio_data/base/templates/accounts/login_base.html

```diff
@@ -20,19 +20,19 @@
 {%- from "_formhelpers.html" import render_field with context %}
 {%- extends "page.html" %}
 {%- set title = None %}
-{%- set activated_providers = config.CFG_OPENID_PROVIDERS * config.CFG_OPENID_AUTHENTICATION
-                           + config.CFG_OAUTH1_PROVIDERS * config.CFG_OAUTH1_AUTHENTICATION
-                           + config.CFG_OAUTH2_PROVIDERS * config.CFG_OAUTH2_AUTHENTICATION %}
+{%- set activated_providers = config.get('CFG_OPENID_PROVIDERS', []) * config.CFG_OPENID_AUTHENTICATION
```

Comment:
Can you check if we still need to override the `login_base.html`? It should be enough to modify only `login.html`.